scripts/release-notes.py

```diff
@@ -87,7 +88,12 @@ def main(token, repo, num_commits, exclude_dependabot):
     other_results = []
     commits_with_multiple_labels = []
 
+    progress = 0
     for commit in commits:
+        if not (progress % 10):
+            print(f"Processing commit {progress + 1}")
+        progress = progress + 1
+
```

Comment:
How about this,
![s3gpe43u](https://github.com/jaegertracing/jaeger/assets/94157520/05897274-d72b-4c38-8c37-80248ba366b0)

Reply:
what does it take to do that?

Reply:
Not any of the external libs. 

```
progress_iterator = 0 # To be declared before commit in commits loop
```
```py
        progress = (progress_iterator + 1) / num_commits
        percentage = progress * 100
        progress_iterator += 1
        # Update the loading bar
        sys.stdout.write('\r[' + '='*int(progress*50) + ' '*(50-int(progress*50)) + f'] {percentage:.2f}%')
        sys.stdout.flush()
```

Reply:
neat, feel free to add. I would put it in a function, so that you can call it like this:
```
progress = updateProgress(progress, num_commits)
```